topics/spacetraders/index.md

```diff
@@ -0,0 +1,9 @@
+---
+topic: spacetraders
+aliases: spacetraders-api, spacetraders-client, spacetraders-gui, spacetraders-v2
+display_name: SpaceTraders
+related: video-games
+short_description: SpaceTraders is an API-based game where you acquire and manage a fleet of ships to explore, trade, and fight your way across the galaxy.
```

Comment:
short_description: SpaceTraders is an API-based game where you manage a fleet of ships to explore, trade, and f ight across the galaxy.
```suggestion
short_description: SpaceTraders is an API-based game where you acquire and manage a fleet of ships to explore, trade, and fight your way across the galaxy.
```